scripts/paycontract.py

```diff
@@ -0,0 +1,33 @@
+def pay(pay, balance1, balance2):
+    balance1 = int(balance1)
+    balance2 = int(balance2)
+    pay = int(pay)
```

Comment:
[optional] could compact it by assigning on one line. Just style preference
```suggestion
    balance1, balance2, pay = int(balance1), int(balance2), int(pay)
```